controllers/tools/applications.js

```diff
@@ -202,14 +195,22 @@ function addCountry(row) {
     return data;
 }
 
+function getApplicationTitle(appId) {
+    let title;
+    if (appId === 'awards-for-all') {
+        title = 'National Lottery Awards for All';
+    } else {
+        title = 'Your funding proposal';
+    }
+    return title;
+}
+
 function getFeedbackDescriptionByAppId(appId) {
     let description;
-    switch (appId) {
-        case 'awards-for-all':
-            description = 'National Lottery Awards for All';
-            break;
-        default:
-            break;
+    if (appId === 'awards-for-all') {
+        description = 'National Lottery Awards for All';
```

Comment:
Is this just the same mapping as `getApplicationTitle()` above?

Reply:
It is, but accidentally I think, so made them separate functions.

Reply:
Oh, on second thoughts it looks like the feedback description _is_ the form title. I can probably merge these then